Dockerfile

```diff
@@ -1,14 +1,26 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
+
+ENV VPNADDR \
+    VPNUSER \
+    VPNPASS
+
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-  apt-get install -y expect wget net-tools iproute ipppd iptables ssh curl && \
-  rm -rf /var/lib/apt/lists/*
+  apt-get install -y -o APT::Install-Recommends=false -o APT::Install-Suggests=false \
+  expect \
+  net-tools \
+  iproute2 \
+  ipppd \
+  iptables \
+  && apt-get clean -q && apt-get autoremove --purge \
+  && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /root
 
 # Install fortivpn client unofficial .deb
-RUN wget 'https://hadler.me/files/forticlient-sslvpn_4.4.2329-1_amd64.deb' -O forticlient-sslvpn_amd64.deb
-RUN dpkg -x forticlient-sslvpn_amd64.deb /usr/share/forticlient
+COPY forticlient-sslvpn_amd64.deb .
+RUN dpkg -x forticlient-sslvpn_amd64.deb /usr/share/forticlient && rm forticlient-sslvpn_amd64.deb
```

Comment:
I don't see the point to this as the file will still be present in the previous layer, which will count toward the image size.

If you're looking to save some storage space, AFIK the only options are:
 - Downloading the file and deleting it on the same `RUN` command (see previous comment)
 - [Multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) (add a lot of complexity)
 - Buildkit's [new features](https://docs.docker.com/develop/develop-images/build_enhancements/) (still experimental)

Then again the ~5Mb of the package might not event be worth the effort.

Reply:
You are right! I didn't know about the build new features you mentioned. Pretty interresting thought.
I would suggest to use the multi-stage builds but as you mentioned earlier, It doesn't worth the effort.